src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -5,42 +5,65 @@
     justify-content: space-between;
     background-color: $ez-ground-primary;
     align-items: center;
-    padding: .5rem 1rem;
+    padding: 0 1.25rem;
+    height: 50px;
+    border-radius: .25rem .25rem 0 0;
 
-    h5 {
+    p {
```

Comment:
I know you just change `h5` to `p` but we should avoid tag selectors. Could you change it to class: `ez-table-header__text`?

Reply:
I would name it to: `ez-table-header__headline`